interface/globals.php

```diff
@@ -209,11 +209,22 @@ function undoMagicQuotes($array, $topLevel=true) {
         }
       }
     }
-    if ($gl_name == 'language_menu_other') {
+    if ($gl_name == 'language_menu_other') {       
       $GLOBALS['language_menu_show'][] = $gl_value;
     }
     else if ($gl_name == 'css_header') {
       $GLOBALS[$gl_name] = "$rootdir/themes/" . $gl_value;
+      
+      // For RTL languages, substitute the same theme converted for RTL// 
+      if ( isset( $_SESSION['language_direction'] ) &&
+              $_SESSION['language_direction'] == 'rtl' && 
+              !strpos($GLOBALS['css_header'], 'rtl')  ) {
+        $GLOBALS['css_header'] = "$rootdir/themes/rtl_" . $gl_value; // str_replace( 'themes/style_', 'themes/rtl_style_', $GLOBALS['css_header'] );
```

Comment:
1. To keep same format as above us $gl_name rather than css_header for index in $GLOBALS.

2. Check to ensure the "converted" rtl file does exist:
If it does exists, then do above.
If it doesn't exists, then throw an error via error_log(something like "Missing the rtl companion theme") and then resort to the original theme above.